M2/Macaulay2/packages/SpectralSequences.m2

```diff
@@ -1546,325 +1476,111 @@ doc ///
 
 doc ///
     Key
-      "Identifying anti-podal points of the two sphere"
+      "Spectral sequences and non-Koszul syzygies"
```

Comment:
Is there just some reordering of the documentation nodes happening here? The differences looks very large so it's hard to tell.